agent/engine/docker_task_engine.go

```diff
@@ -46,6 +47,8 @@ const (
 	capabilityTaskIAMRole        = "task-iam-role"
 	capabilityTaskIAMRoleNetHost = "task-iam-role-network-host"
 	labelPrefix                  = "com.amazonaws.ecs."
+	ECSDataDir                   = "/data/"
+	HostDataDir                  = "/var/lib/ecs/"
```

Comment:
My bad, I've already moved them to the config package but forgot to remove them here (Or they came back when I did a conflict merge). These constants have no actual effect.